frontend/src/Styles/Timer.module.css

```diff
@@ -0,0 +1,13 @@
+.time{
+    margin-bottom: 1rem;
+}
+
+.time span{
+    font-size: 3rem;
+}
```

Comment:
Could now be one if you merge the span's into one line.

Reply:
Thanks a lot. I fixed it.